.luacheckrc

```diff
@@ -0,0 +1,35 @@
+std             = "lua51c+lua52c+lua53c+luajit+busted"
```

Comment:
`busted` shoudn't be included and without it the line can be removed (latest Luacheck uses `std = max` by default and that is equivalent to `lua51c+lua52c+lua53c+luajit`).